packages/scandipwa/src/component/ProductCard/ProductCard.container.js

```diff
@@ -120,7 +120,7 @@ export class ProductCardContainer extends PureComponent {
 
         // if 'Product Use Categories' is enabled then use the current window location to see if the product
         // has any url_rewrite for that path. (if not then just use the default url)
-        const rewriteUrl = url_rewrites.find(({ url }) => url.includes(productUrl)) || {};
+        const rewriteUrl = url_rewrites?.find(({ url }) => url.includes(productUrl)) || {};
```

Comment:
Should here be this `?`

Reply:
You are right. Default values are much better. Fixed.